src/applications/vaos/newAppointmentFlow.js

```diff
@@ -67,8 +67,8 @@ export default {
           // Check if user registered systems support community care...
           const userSystemIds = await getSystemIdentifiers();
           const ccSites = await getSitesSupportingVAR();
-          const ccEnabledSystems = userSystemIds.filter(id =>
-            ccSites.some(site => site._id === id),
+          const ccEnabledSystems = userSystemIds.filter(uniqueId =>
```

Comment:
Should this be changed? It was the only way I could see to get it to work.

Reply:
This is fixed in master, I think you'll want to remove this change and pull in the latest from master.